link_test.go

```diff
@@ -1899,64 +1901,56 @@ func TestLinkAltName(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	err = LinkAddAltName(link, "altname")
-	if err != nil {
-		t.Fatalf("Could not add altname: %v", err)
-	}
+	altNames := []string{"altname", "altname2", "some_longer_altname"}
 
-	err = LinkAddAltName(link, "altname2")
-	if err != nil {
-		t.Fatalf("Could not add altname: %v", err)
+	for _, altname := range altNames {
+		err = LinkAddAltName(link, altname)
+		if err != nil {
+			t.Fatalf("Could not add %s: %v", altname, err)
+		}
 	}
 
 	link, err = LinkByName("bar")
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	altNameExist := false
-	altName2Exist := false
-	for _, altName := range link.Attrs().AltNames {
-		if altName == "altname" {
-			altNameExist = true
-		} else if altName == "altname2" {
-			altName2Exist = true
-		}
+	sort.Strings(link.Attrs().AltNames)
+	sort.Strings(altNames)
```

Comment:
can you move this right after the definition of `altNames` ?

Reply:
Done